root/static/scripts/common/components/TagEditor.js

```diff
@@ -362,16 +362,21 @@ class TagEditor extends React.Component<TagEditorProps, TagEditorState> {
 
   updateTags(updatedUserTags: $ReadOnlyArray<TagUpdateT>) {
     const newTags = this.state.tags.slice(0);
-
```

Comment:
Sometimes blank lines are removed like here, but otherwise added like in https://github.com/metabrainz/musicbrainz-server/pull/1062/files#diff-0f30970e52953a6ba4194b9157fe8b4b and I'm not sure I see an improvement (or un-improvement) either way, so I think it'd be nicer to keep the diff clean, but doesn't matter much at this point anyway :)

Reply:
I removed the new line on that file at least, since that was the only change to that file apparently in the end, and readded it here, since it seems sensible to have.